src/octoprint/timelapse.py

```diff
@@ -838,7 +839,40 @@ def _render(self):
 		with self.render_job_lock:
 			try:
 				self._notify_callback("start", output)
-				p = sarge.run(command_str, stdout=sarge.Capture(), stderr=sarge.Capture())
+				p = sarge.run(command_str, stdout=sarge.Capture(), stderr=sarge.Capture(), async=True)
+
+				self._logger.debug("Parsing ffmpeg output")
+
+				# Compile regex to make it faster
```

Comment:
Could you add an example output line that this will parse, for future reference? Would help maintenance down the road tremendously :)